csrc/driver_api.h

```diff
@@ -84,16 +85,41 @@ namespace nvfuser {
 #endif
 
 #if (CUDA_VERSION >= 12000)
-#define NVF_DRIVER_API_WRAPPER_CUDA_120(fn) fn(cuTensorMapEncodeTiled, 12000)
+#define NVF_DRIVER_API_WRAPPER_CUDA_120(fn)  \
```

Comment:
https://github.com/NVIDIA/Fuser/pull/5433 dropped CUDA <12 so you can now remove some version checks.